### ISSUE TITLE
Push Notifications


### ISSUE THREAD

**Issue by ankitmodanwall** [mansiruhil/fail-u-forward#193]
**Description**:
- [ ] Enable real-time push notifications so users are instantly alerted when:

- [ ] Someone follows them

- [ ] Someone likes/claps their post

- [ ] Someone comments or replies to their comment

**Comment by Atomic-Shadow7002** [comment]
Hey, can you please assign this issue to me?
